src/mac/mac_bs.c

```diff
@@ -692,6 +709,19 @@ void *mac_bs_tap_rx_th(void *arg) {
       MacDataFrame frame = dataframe_create(dev->bytes_rec);
       memcpy(frame->data, dev->buffer, dev->bytes_rec);
 
+      // Packet inspection: determine if this is TCP traffic
```

Comment:
Also used in mac_ue.c -> move this functionality to a common place?

Reply:
This is now moved to mac_common.c